qemu/tests/cfg/virt_firmware_basic_test.cfg

```diff
@@ -0,0 +1,11 @@
+- virt_firmware_basic_test:
+    no i440fx
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9.u0 Host_RHEL.m9.u1
```

Comment:
However, virt-firmware can be installed by users using [pip install virt-firmware](https://pypi.org/project/virt-firmware/), so I think this line is not suitable, the virt-firmware should not link to any distro.